session/sessionstate.go

```diff
@@ -988,3 +992,20 @@ func (state *State) ClearSubscribedObjects(IDs []string) error {
 func (state *State) UpdateFeatureMap(host string, actionState *action.State) {
 	state.Features.UpdateFeatureMap(state.Rest, host, actionState, state.LogEntry)
 }
+
+// AddCustomState add custom data handler to state object
+func (state *State) AddCustomState(key string, value interface{}) {
```

Comment:
Don't really understand what these are supposed to be used for

Reply:
When you need to add data propagated between actions when using qlik-oss/gopherciser as a library

Reply:
Between actions??? Crazy